test/unit/resources/legacy_gelamtest.js

```diff
@@ -0,0 +1,351 @@
+/**
+ * LegacyGelamTest: Supporting actor-style tests after it was cool.
```

Comment:
Hah, well, just remember all of the logging and testing was designed to be able to get realtime performance/debug info from and also test a federated messaging system consisting of both clients and servers running in distinct processes and potentially in distinct machines (eventually).  So its goals were somewhat more overkill than GELAM.  (And didn't have the benefit of experience with the ergonomics of writing such tests, dealing with the growth of the system, or ramping up additional contributors.)

I am very happy to see our bright new future come into being!